docs/products/storage/backups/guides/restore-to-an-existing-instance/index.md

```diff
@@ -33,14 +33,22 @@ This process restores all data that was stored on the disk at the time the backu
 
 1.  From the **Restore to Existing Linode** menu, click **Restore**. Your backup will begin restoring to your Compute Instance, and you can monitor its progress in the notifications area. Note that the time it takes to restore your backup will vary depending upon the restore size and the number of files being restored.
 
     {{< note >}}
     If you are attempting to restore a disk to the same Compute Instance the backup was created from, the restoration process will not delete the original disk for you. Manually delete the original disk to make room for the backup, if desired.
     {{< /note >}}
 
-1.  Once your backup has been restored, you can optionally power up or restart your Compute Instance using that backup. To do so, navigate to the target Compute Instance in the Cloud Manager, go to the **Configurations** tab, locate the configuration profile created during the restore process, and click the corresponding **Boot** button. For full instructions, review [Manage Configuration Profiles on a Compute Instance > Boot from a Configuration Profile](/docs/products/compute/compute-instances/guides/configuration-profiles/#boot-from-a-configuration-profile).
\ No newline at end of file
+1.  Once your backup has been restored, you can optionally power up or restart your Compute Instance using that backup. To do so, navigate to the target Compute Instance in the Cloud Manager, go to the **Configurations** tab, locate the configuration profile created during the restore process, and click the corresponding **Boot** button. For full instructions, review [Manage Configuration Profiles on a Compute Instance > Boot from a Configuration Profile](/docs/products/compute/compute-instances/guides/configuration-profiles/#boot-from-a-configuration-profile).
+
+    {{< note type="warning" title="Warning: UUID Collisions">}}
+    When you restore a backup, the restored disk is assigned the same [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier) as the original disk. In most cases, this is acceptable and does not cause issues. However, if you attempt to mount both the original disk and the corresponding restore disk at the same time (by assigning them both to devices in your Configuration Profile's **Block Device Assignment**), you will encounter a UUID "collision".
+
+    When this happens, the system selects, and mounts, only one of the disks at random. This is due to both disks sharing the same UUID, and your instance *may fail to boot* since it will not be clear which disk is root. If your system does boot, you will not see any immediate indication if you are booted into the restored disk or the original disk, and you will be unable to access both disks at the same time.
```

Comment:
Approving this PR, with the condition that the following is true:

> This is due to both disks sharing the same UUID, and your instance *may fail to boot* since it will not be clear which disk is root. If your system does boot,...

If it's not true, tag me in again once there's a copy edit.